### PR TITLE
adds frostbite spiders and ice watchers to iceplanet

### PR DESCRIPTION
**Pull request by Marmio64**
### Intent of your Pull Request
adds frostbite spiders and ice watchers to ice planet.
Frostbite spiders are marrow weavers that deal slightly less damage but inject frost oil with each bite.
Both have very rare chances to spawn, only a bit higher than a spawner, and half the chance of goldgrub spawns.
edit: the frostbite spiders now have the same chance to spawn as the goldgrub, and also get a unique model.
iceweaver

### Why is this change good for the game?
more iceplanet fauna good

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
adds frostbite spiders and ice watchers to iceplanet

### What should players be aware of when it comes to the changes your PR is implementing?
adds frostbite spiders and ice watchers to iceplanet

### What general grouping does this PR fall under? 
For example, Engineering changes, Medical rebalancing, TEG tweaks, etc.?
mining changes

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
frostbite spiders deal a couple points less damage than marrow weavers.

:cl:  
rscadd: Adds frostbite spiders and puts them in iceplanet along with icewatchers
/:cl:

### CODE REVIEW

**Comment by ChesterTheCheesy** [comment]
Cmon, the new weaver just asks for a cool chilly sprite!

**Comment by Marmio64** [comment]
> Cmon, the new weaver just asks for a cool chilly sprite!

make me one and ill give it the sprite
otherwise it cannot happen due to my lack of artistic talent

**Comment by ChesterTheCheesy** [comment]
no artistic talent required
obraz

https://drive.google.com/file/d/13f9dlCEZ49-mFds-smi7FwG1yfJukL8y/view?usp=sharing

**Review by nesisknesis — APPROVED** [review]
